spec/factories/activity.rb

```diff
@@ -22,12 +22,18 @@
 
     association :organisation, factory: :organisation
 
-    factory :programme_activity do
-      level { :programme }
-    end
-
     factory :fund_activity do
       level { :fund }
+      funding_organisation_name { "HM Treasury" }
+      funding_organisation_reference { "GB-GOV-2" }
+      funding_organisation_type { 10 }
```

Comment:
Same point about using an integer for a string type in the factories. Think it would be clearer to change this to "10"?